e2etest/e2etest_suite_test.go

```diff
@@ -146,16 +146,19 @@ var _ = ginkgo.BeforeSuite(func() {
 	framework.ExpectNoError(err)
 
 	_, err = cs.CoreV1().ConfigMaps(testNameSpace).Get(context.TODO(), configMapName, metav1.GetOptions{})
-	framework.ExpectEqual(errors.IsNotFound(err), true)
-
-	// Init empty MetalLB ConfigMap.
-	_, err = cs.CoreV1().ConfigMaps(testNameSpace).Create(context.TODO(), &v1.ConfigMap{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      configMapName,
-			Namespace: testNameSpace,
-		},
-	}, metav1.CreateOptions{})
-	framework.ExpectNoError(err)
+	// When running in operator mode the configmap can be found in the cluster which is a valid state.
```

Comment:
Can we just check if we are running in operator mode and skip?

Reply:
we need to create the configmap if that's the first run. it's not created in the deployment

Reply:
Even if we are running in operator mode?
What I am suggesting here is to condition the notFoundCheck / configmap creation only when we are not running in operator mode.

Reply:
yeah i get it, but if we deploy metallb with the operator, what would create the configmap?

Reply:
The operator?

Reply:
ok sorry i get it now